pkg/machines/components/vmDisksTab.jsx

```diff
@@ -186,7 +186,7 @@ class VmDisksTab extends React.Component {
                 );
                 columns.push({ title: diskActions });
             }
-            return { columns };
+            return { columns, id: disk.target };
```

Comment:
Passing a `key:` (or `rowKey:`) here would IMHO make more sense. keys can have a much more flexible content than IDs.

Reply:
Passing a rowKey to the row properties won't make PF use it unfortunately. 
I need to write some clean reproducer and submit an issue to PF, because the current way to have custom row keys (something else than index) is not very nice actually :/

Reply:
Check my updated PR for implementation details.